lib/portage/gpkg.py

```diff
@@ -778,10 +778,16 @@ def __init__(self, settings, base_name=None, gpkg_file=None):
             self.gpkg_file = _unicode_decode(
                 gpkg_file, encoding=_encodings["fs"], errors="strict"
             )
-        self.base_name = base_name
+
+        if basename:
```

Comment:
We really prefer explicit `is None` if you don't expect any other non-true values.

Reply:
I thought the basename could be empty "" if it does not know what the package is, but the caller could set it to None. So going to use `is None`.